analyze.go

```diff
@@ -108,19 +108,17 @@ func analyze(fname string, stmt *Directive, term string, ctx blockCtx, options *
 		}
 	}
 
-	// If DirectiveSources was not provided, try to find the directive in legacy way.
-	// We want to use DirectiveSources to indicate the OSS/N+ version, and dynamic
-	// modules we want to include for validation. However, invokers used MatchFuncs
-	// and a directive map before. This is a transition plan. After MatchFuncs is deleted
-	// from ParseOptions, this if block should be deleted.
+	// If DirectiveSources was not provided, use the default map
```

Comment:
"use the default map" sounds like a direction to the reader. What I think you mean to communicate is that if the `DirectiveSources` option is not provided (is empty?) then the parser will match against a default set of directives.